setup_build.py

```diff
@@ -20,9 +20,11 @@ def walk_dep_tree(system):
     statics that the system depends on.
 
     system is a dict containing:
-        {'packages': ['Package'],
-         'systems':['foo'],
-         'statics': ['bar']}
+        {'bin': ['bin_script_A'],
```

Comment:
Honestly, I have no idea what `bin_script_A`, etc. means in this context. Therefore, at least I suggest to add proper doc string about to explain this notations.